modules/gapi/src/compiler/gcompiler.cpp

```diff
@@ -515,7 +522,11 @@ cv::GCompiled cv::gimpl::GCompiler::compile()
 {
     std::unique_ptr<ade::Graph> pG = generateGraph();
     runPasses(*pG);
-    compileIslands(*pG);
+    if (!m_metas.empty())
```

Comment:
Why do you need this check?

Reply:
According to the `streamingCompiled` not to compile graph without meta